### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by alexdanilowicz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,24 +4,24 @@ NPFISH is an experimental game where you can code your own Chess AI that will be
...
-1. Create a new file in the `engines` folder
-2. Copy the code in `TemplateBot.ts`
-3. Rename the class to your own bot name, and fill in the `name` variable
-4. Add your bot to the array in `engines/types.ts`
-5. Go ahead and code your own logic in your new file!
-6. You can see `BasicMinimax` as an example
+1. Run `yarn` in root to install dependencies.
+1. Create a new file in the `engines` folder. You could call it `[your_name]Bot.ts`. For example, `TeddyBot.ts`.
+1. Copy the code in `TemplateBot.ts`
+1. Do the TODOs. For example, change the clas name to your own bot name, and fill in the `name` variable
```
Comment (alexdanilowicz) [inline README.md]:
```suggestion
1. Do the TODOs. For example, change the class name to your own bot name, and fill in the `name` variable
```

File: README.md [hunk 2]
```diff
@@ -4,24 +4,24 @@ NPFISH is an experimental game where you can code your own Chess AI that will be
...
+1. Add your bot to the array in `engines/types.ts`
+1. Now, code your own logic in the file you created!
 
-There is a default Minimax algorithm function in the `engines/utils.ts` file that you are welcome to copy, if you just want to play around with the board evaluation functions.
+> Hint: look at `BasicMinimax` in the engines directory as an example. You are welcome to copy it if you just want to test it out
```
Comment (alexdanilowicz) [inline README.md:15]: Need to add objective and what to do here. Like step 1 should be writing a function that decides a move and returns an evaluation. Step 2 should be writing an evaluation function.

File: README.md [hunk 3]
```diff
@@ -4,24 +4,24 @@ NPFISH is an experimental game where you can code your own Chess AI that will be
...
 
-Some hints...
+Some caveats...
```
Comment (alexdanilowicz) [inline README.md]: very confused why these are called hints. feels like more like a thought? idk i was confused why "code is not written in optimal way" is a hint

File: engines/TemplateBot.ts [hunk 4]
```diff
@@ -28,9 +31,11 @@ export class TemplateBot extends Engine {
...
 
-  // This calculates the maximum time in milliseconds your engine should think for
-  // If onFinish is called first, your engine will not think for this long
-  // If this time hits, then your engine will return the latest onUpdateResponse call
+  /*
+   * Calculates the maximum time in milliseconds your engine should think for
+   * If onFinish is called first, your engine will not think for this long
+   * If this time hits, then your engine will return the latest onUpdateResponse call
+   */
   getCalculationTimeInMs(board: Chess): number {
```
Comment (alexdanilowicz) [inline engines/TemplateBot.ts:39]: need more jsx comments in the engines file

File: engines/types.ts [hunk 5]
```diff
@@ -10,4 +11,4 @@ export type EngineResponse = {
   numberOfNodesSearched: number
 }
 
-export const Engines = [RandomBot, BasicMinimaxD23S, TeddyBot]
+export const Engines = [RandomBot, BasicMinimaxD23S, TeddyBot, AlexBot]
```
Comment (alexdanilowicz) [inline engines/types.ts]:
```suggestion
export const Engines = [RandomBot, BasicMinimaxD23S, TeddyBot]
```

File: engines/types.ts [hunk 6]
```diff
@@ -1,5 +1,6 @@
 import { Move } from 'chess.js'
 
+import { AlexBot } from './AlexBot'
```
Comment (alexdanilowicz) [inline engines/types.ts]:
```suggestion
```

File: README.md (hunk 3 above)
Comment (Teddarific) [reply README.md]: hints as to how you can improve this bot

**Pull request merged**